src/main/java/net/javadiscord/javabot/data/config/guild/MetricsConfig.java

```diff
@@ -11,8 +11,8 @@
 @Data
 @EqualsAndHashCode(callSuper = true)
 public class MetricsConfig extends GuildConfigItem {
-	private String weeklyMessages = "";
-	private String activeMembers = "";
+	private long weeklyMessages = -1;
+	private long activeMembers = -1;
```

Comment:
Side question: why are these initialized with -1 rather than 0?

Reply:
If they are -1,it can clearly be differentiated as "no value present" asopposed to "no messages".